trieste/models/model_interfaces.py

```diff
@@ -624,22 +624,37 @@ def randomize_hyperparameters(object: gpflow.Module) -> None:
             param.assign(param.prior.sample())
 
 
-def squeeze_hyperparameters(object: gpflow.Module, alpha: float = 1e-2) -> None:
+def squeeze_hyperparameters(
+    object: gpflow.Module, alpha: float = 1e-2, epsilon: float = 1e-7
+) -> None:
     """
-    Squeezes the parameters to be strictly inside their range defined by the Sigmoid.
+    Squeezes the parameters to be strictly inside their range defined by the Sigmoid,
+    or strictly greater than the limit defined by the Softplus.
     This avoids having Inf unconstrained values when the parameters are exactly at the boundary.
 
     :param object: Any gpflow Module.
-    :param alpha: the proportion of the range with which to squeeze
-    :raise ValueError: If ``alpha`` is not in (0,1).
+    :param alpha: the proportion of the range with which to squeeze (or the absolute value
+    with which to translate the parameter for Softplus without lower bound)
```

Comment:
Is the code actually handling the case of a Softplus without a lower bound?

Reply:
no, the transform has to be shift + softplus, I only coded that case because it's the `positive` transform used in gpflow. But you have cases where the shift is zero.